metaflow/plugins/__init__.py

```diff
@@ -9,8 +27,15 @@ def get_plugin_cli():
     from . import package_cli
     from .aws.batch import batch_cli
 
-    return [package_cli.cli,
-            batch_cli.cli]
+    return ext_plugins.get_plugin_cli() + [package_cli.cli, batch_cli.cli]
+
+
+def _merge_lists(decos, overrides, attr):
```

Comment:
Could you rename `decos` to something less-decorator specific. It seems you use this function for `environments` as well.
Maybe base and overrides?

Reply:
Done